DZSwiftUI/DZSwiftUI/PlayerView.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  PlayerView.swift
+//  DZSwiftUI
+//
+//  Created by Maxim on 18.01.2023.
+//
+
+import SwiftUI
+import AVFoundation
+
+/// Плеер
+final class PlayerView: ObservableObject {
+    // MARK: - Private Constants
+    
+    private enum Constants {
+        static let maxDurationStart = 0.0
+        static let songIndexStart = 0
+        static let currentDurationStart = 0.0
+        static let songAuthorNames = ["Макс Корж", "Noize MC"]
+        static let songNames = ["Жить в кайф", "Вояджер 1"]
+        static let albumImageNames = ["coolLife", "voyadger"]
+        static let timerTimeInterval = 1
+        static let audioType = "mp3"
+        static let errorText = "Ошибка:"
+    }
+    
+    // MARK: - Public Properties
+    
+    @Published var currentDuration = Constants.currentDurationStart
+    @Published var maxDuration = Constants.maxDurationStart
+    var albumImageNames = Constants.albumImageNames
```

Comment:
```suggestion

    var albumImageNames = Constants.albumImageNames
```
отделяй враперы от обычных свойств